### ISSUE TITLE
Remote path has movies and tvshows in m3u

### ISSUE THREAD

**Issue by Viraagj** [kodi-pvr/pvr.iptvsimple#719]
Hi,
I have a IPTV service that has movies and tv shows in it 

But IPTV client keeps them as a channel 

Is there a way to shift them in tv shows and movies tab respectively

**Comment by phunkyfish** [comment]
No, not really. PVR is live TV so doesn’t really have that distinction.

The best you can do is have it as TV and Recordings. In iptvsimple anything that is VOD can be classed as Recording, I.e. pre-recorded media.

In iptvsimple recordings are ready only and cannot be created by the addon.

**Comment by Viraagj** [comment]
Ok

**Comment by Viraagj** [comment]
Thanks
